core/src/main/java/io/substrait/type/proto/BaseProtoTypes.java

```diff
@@ -41,6 +48,13 @@ public BaseProtoTypes(Type.Nullability nullability) {
     INTERVAL_DAY = wrap(Type.IntervalDay.newBuilder().setNullability(nullability).build());
     INTERVAL_YEAR = wrap(Type.IntervalYear.newBuilder().setNullability(nullability).build());
     UUID = wrap(Type.UUID.newBuilder().setNullability(nullability).build());
+    VARCHAR = wrap(Type.VarChar.newBuilder().setNullability(nullability).build());
+    FIXED_CHAR = wrap(Type.FixedChar.newBuilder().setNullability(nullability).build());
+    FIXED_BINARY = wrap(Type.FixedBinary.newBuilder().setNullability(nullability).build());
+    DECIMAL = wrap(Type.Decimal.newBuilder().setNullability(nullability).build());
```

Comment:
Decimal also has precision (and scale). Should we have a method for creating a decimal instead?